### DISCUSSION TITLE
Pangeo Showcase talk on xbitinfo?

### DISCUSSION THREAD

**Discussion by rsignell-usgs** [observingClouds/xbitinfo#112] [General]
@xbitinfo folks, we are going to be starting up the Pangeo Showcase talks again at
https://pangeo.io/pangeo-showcase.html
these are 15 min talks that get recorded and get a DOI, and it's a great way to connect/engage with a bunch of folks excited about new approaches to big data. 

It would be very cool to get a talk on where you are (and what your plans are) for xbitinfo!

What do you think?

**Comment by observingClouds** [comment]
This sounds like a great opportunity! Thanks for the suggestion @rsignell-usgs! Late August will be the first opportunity for me to present this work though. I'll keep in mind.

**Comment by rsignell-usgs** [comment]
@observingClouds ,  I hope you are still interested!  Could we schedule something for September or October?

**Reply by observingClouds** [reply]
Thanks @rsignell-usgs for staying in touch. I just filled out [this form](https://forms.gle/QwxKusVvrvDakSNs8). Wednesdays at 4PM EST would work best for me. Mid-September to October should work.

**Comment by observingClouds** [comment]
The talk is now scheduled for October 26th! Zoom details and abstract can be found at https://discourse.pangeo.io/t/wednesday-october-26th-2022-xbitinfo-compress-datasets-based-on-their-information/2830

**Comment by observingClouds** [comment]
The slides of the talk are available at
[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.7259092.svg)](https://doi.org/10.5281/zenodo.7259092)
and the recording can be viewed on [YouTube](https://www.youtube.com/watch?v=oowb5bxWQAw)
